migrate/v0_14/migrate.md

```diff
@@ -1,13 +1,13 @@
-# Kava-5 Upgrade Instructions
+# Kava-5.1 Upgrade Instructions
 
 ## Software Version and Key Dates
 
-* The version of Kava for kava-5 is v0.13.1
-* Kava-4 validators should prepare to shutdown their nodes March 4th, 2021 at 13:00 UTC
-* Kava-5 genesis time is set to March 4th, 2021 at 15:00 UTC
-* The version of cosmos-sdk for kava-5 is v0.39.2
-* The version of tendermint for kava-5 v0.33.9
-* The minimum version of golang for kava-5 is 1.13+, 1.15+ has been tested and is recommended.
+* The version of Kava for kava-5.1 is v0.14.0
+* Kava-6 validators should prepare to shutdown their nodes March 24th, 2021 at 13:00 UTC by setting `--halt-time` to `1616590800`
```

Comment:
Going from version 6 to 5.1 is unintuitive, let's add a note that kava-6 refers to the currently running mainnet chain and kava-5.1 refers to the chain we'll be upgrading to.